ios/.gitattributes

```diff
@@ -0,0 +1 @@
+*.pbxproj merge=union
```

Comment:
See http://roadfiresoftware.com/2015/09/automatically-resolving-git-merge-conflicts-in-xcodes-project-pbxproj/ for reference.